src/component/requests.js

```diff
@@ -15,32 +17,69 @@ export default class Requests extends React.Component {
     });
   }
 
+  componentDidMount() {
+    const requests = React.findDOMNode(this);
```

Comment:
Very valid point. To be honest I already thought about renaming this constant already